tests/network/vmi_networking.go

```diff
@@ -879,9 +859,9 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				podIP := libnet.GetPodIPByFamily(virtHandlerPod, k8sv1.IPv4Protocol)
 				Expect(ping(podIP)).To(Succeed())
 
-				By("Execute migration")
+				By("starting the migration")
```

Comment:
nit:
```suggestion
				By("Starting the migration")
```

Reply:
it's used both with/without capital all over [migration_test.go ](https://github.com/kubevirt/kubevirt/blob/cd3239c5185f8b1e5dfcf5feeb14b853b347e7de/tests/migration_test.go). Let's fix it together on another PR